frontend/src/components/loaders/LoadingSpinner.jsx

```diff
@@ -0,0 +1,7 @@
+const LoadingSpinner = () => (
+  <div className='flex justify-center items-center h-64'>
```

Comment:
Will do

Reply:
I don't see the loading spinner on the style page. Are we cancelling this for now?

My thought was also...if we want to incorporate custom design into components to do a while label-able version of SocialPredict with different colors and about page, it might be nice to have a style page that an admin can reference to ensure their design looks good.

Reply:
> I don't see the loading spinner on the style page. Are we cancelling this for now?

Added https://github.com/openpredictionmarkets/socialpredict/pull/242/commits/e1a6c0b175dc0da6b67b31a65a1d0ac002b53819